### PR TITLE
BufferedInputStream enhancement android

### PR DESCRIPTION
**Pull request by Scrappers-glitch**
Targeting issue #1622

### CODE REVIEW

**Comment by Scrappers-glitch** [comment]
@Ali-RS Hello, 

if you have already tested the `AndroidBufferImageLoader` and its working fine, then no need to change it, and this PR would be ready to be merged.

**Comment by Ali-RS** [comment]
Thanks. 
Going to merge this in next 24H.

File: jme3-android/src/main/java/com/jme3/texture/plugins/AndroidBufferImageLoader.java [hunk 1]
```diff
@@ -158,8 +153,7 @@ public Object load(AssetInfo assetInfo) throws IOException {
         data.flip();
         
         bitmap.recycle();
-        
-        Image image = new Image(format, width, height, data, ColorSpace.sRGB);
-        return image;
+       
+        return new Image(format, width, height, data, ColorSpace.sRGB);
```
Comment (stephengold) [inline jme3-android/src/main/java/com/jme3/texture/plugins/AndroidBufferImageLoader.java:157]: I prefer the old code, which assigned the new object to a variable and returned the variable. That made it easy to examine the new object in the debugger.

Comment (Scrappers-glitch) [reply jme3-android/src/main/java/com/jme3/texture/plugins/AndroidBufferImageLoader.java]: Changes Applied

**Comment by stephengold** [comment]
My thanks to you both!

**Pull request merged**